Github Actions tests broken

```
E   ValueError: greenlet.greenlet size changed, may indicate binary incompatibility. Expected 152 from C header, got 40 from PyObject
```
Appears on all runs in the last few days. Seems to work locally for me. Maybe updating `gevent` will solve this, but I don't know what triggered this since the version hasn't been changed in 2 years and no relevant packages have been updated recently.

--- a/requirements.txt
+++ b/requirements.txt
@@ -20,7 +20,7 @@ pytest==6.2.5
 werkzeug==2.0.1
 redis==3.5.3
 gunicorn==20.1.0
-gevent==20.9.0
+gevent==22.10.2
 pyparsing==2.4.7
 flask-deprecate==0.1.3
 ddt==1.4.4

